### PR TITLE
Character dev

### PR DESCRIPTION
**Pull request by adevgriff**

### CODE REVIEW

File: tnasm/parser.cpp [hunk 1]
```diff
@@ -261,7 +263,7 @@ shared_ptr <token> p_constant_line() {
 }
 
 /*
- * raw_line ::= NUMBER+.
+ * raw_line ::= raw_value.
```
Comment (miniat-amos) [inline tnasm/parser.cpp:266]: should still have the + at the end to signify one or more raw_value.

File: tnasm/parser.cpp [hunk 2]
```diff
@@ -261,7 +263,7 @@ shared_ptr <token> p_constant_line() {
...
  * We'll implement the "one-or-more" NUMBER check manually rather than via the
```
Comment (miniat-amos) [inline tnasm/parser.cpp:267]: You need to update this comment since you're not using NUMBER any more.

File: tnasm/test.asm
Comment (miniat-amos) [inline tnasm/test.asm:1]: Should at least have some escaped charaters

**Review by miniat-amos — CHANGES_REQUESTED** [review]

**Comment by adevgriff** [comment]
Bad formatting settings in vs code killed this PR remaking PR on separate branch. This is being done to keep revision history clean.

**Pull request closed without merging**